### ISSUE TITLE
Calibration (node classification) notebook is outdated

### ISSUE THREAD

**Issue by thatlittleboy** [stellargraph/stellargraph#1365]
Hi there, I was reading through the calibration notebook (`calibration-pubmed-node-classification.ipynb`) and found some errors that might need cleaning up (presented in order from top to bottom):

1. The link to "GraphSAGE on the Cora dataset" notebook in the introduction paragraph is broken.
2. Input Cell [5] has a spelling error in the comments: `epochs = 20  # Numper of training epochs for GraphSAGE model.`
3. More of a question, but in Input Cell [11], shouldn't the target encoding on the validation data be a `.transform` instead of a `.fit_transform`?
4. Under the "Creating the GraphSAGE model in Keras" section, there is a sentence "`... so we choose the GraphSAGENodeMapper as we are predicting node attributes with a GraphSAGE model.`" that references a `GraphSAGENodeMapper` but goes on to use `GraphSAGENodeGenerator`  in the later code cell.
5. Under "Calibration Curves" section, the first paragraph reads:
```
Calibration curves also known as reliability diagrams are a visual method for this task.
See reference [2] for a description of calibration curves also known as reliability diagrams.
```
with some unnecessary information repetition.

6. Input Cell [59] spells "Accuracy" incorrectly as "Accurace".

**Comment by huonw** [comment]
Hi @thatlittleboy, thanks for reading the demo in such detail and taking the time to file an issue. If you're interested, we're very happy to help you submit a pull request adjusting these, otherwise we'll address it some time over the next week.

For the specific points/questions where the reasoning/fix may be unclear:

1\. The correct link is <https://github.com/stellargraph/stellargraph/blob/master/demos/node-classification/graphsage/graphsage-cora-node-classification-example.ipynb>, but better would being a relative link `../node-classification/graphsage/graphsage-cora-node-classification-example.ipynb`

3\. Yeah, `.transform` seems to be the appropriate one there.

4\. 46dfd5e48aae7593dad08b3d3aeff14b3d3cad44 renamed all of the `...Mapper` types to our current `...Generator` ones, but it looks like it missed this aspect of our docs.

**Comment by thatlittleboy** [comment]
> Hi @thatlittleboy, thanks for reading the demo in such detail and taking the time to file an issue.

My pleasure -- I'm just slowly working through the demos to learn more about GCNs and the StellarGraph library.

> If you're interested, we're very happy to help you submit a pull request adjusting these, otherwise we'll address it some time over the next week.

Sure, I'll be happy to make a PR sometime later this week / over the next weekend. Let me take some time to read through the contributing guidelines first :)

**Comment by huonw** [comment]
Given you've opened #1405 (thanks!), I've assigned you to this issue, so it's clearer on our side that someone has picked this up. 😄

**Comment by thatlittleboy** [comment]
Since this has been addressed in #1405 (and merged), I will close the issue now. Thanks.
